packages/tools/release-utils-internal/src/git.ts

```diff
@@ -39,3 +42,37 @@ export async function addRemoteWithAuthentication(repoUrl: string, remoteName: s
 
     await execShellCommand(`git remote add "${remoteName}" "${getGHRepoAuthUrl(repoUrl)}"`);
 }
+
+export function noChanges(): boolean {
+    return /nothing to commit/i.test(sh.exec("git commit --dry-run -uno", { silent: true }));
```

Comment:
Maybe [git status --porcelain](https://stackoverflow.com/a/25149786) is a cleaner option.

Reply:
+1, good point, I'll update